lib/xml.go

```diff
@@ -61,7 +58,7 @@ import (
 //	b"<?xml vers... ...>".decode_xml()   // return { ... }
 //	"<?xml vers... ...>".decode_xml("xsd")   // return { ... }
 //	b"<?xml vers... ...>".decode_xml("xsd")   // return { ... }
-func XML(adapter ref.TypeAdapter, xsd map[string]string) (cel.EnvOption, error) {
+func XML(adapter types.Adapter, xsd map[string]string) (cel.EnvOption, error) {
```

Comment:
This is not a breaking change as `types.Adapter` is [`type Adapter = ref.TypeAdapter`](https://pkg.go.dev/github.com/google/cel-go@v0.20.1/common/types#Adapter).